### DISCUSSION TITLE
Add a Quit Ray option to the tray

### DISCUSSION THREAD

**Discussion by pascalchevrel** [spatie/ray#193] [Ideas]
The tray icon only allows to pop up the Ray window. I would appreciate a Quit Ray option as well

**Comment by freekmurze** [comment]
It should be there, what platform are you on?

<img width="195" alt="image" src="https://user-images.githubusercontent.com/483853/104924180-5b36fd00-599d-11eb-81a6-9824f525ac36.png">

**Comment by pascalchevrel** [comment]
Linux, my tray menu is nothing like the one above:
![Capture d’écran de 2021-01-18 14-58-48](https://user-images.githubusercontent.com/454375/104924834-473fcb00-599e-11eb-9b6a-c7ac770c418c.png)

**Reply by freekmurze** [reply]
@AdrianMrn could you take a look at this?
